docs/2.0/authorization.md

```diff
@@ -253,28 +273,6 @@ end
 
 Now, whatever action you take for one comment, it will be available for the `edit_comments?` method in `PostPolicy`.
 
-## Files
```

Comment:
I think we should somehow specify that the general methods have been removed in 2.31.

I would also leave the old docs specifying that we removed it in 2.31. It's important that folks don't have "dissapearing" sections.

Reply:
Totally agree, it's not the first time I do this mistake. I have a suggestion, thinking long-term I believe that filling the docs with warnings and info about changes along different versions is not scalable enough and we can easily "over-populate" the docs with this warnings. Is there any easy way to organize the docs per version? Lets say I'm using v 2.1 and you v 2.23, when I access the docs would be great to have a drop-down to select my version and see the docs for that version. Most of the content would be the same for all versions but specific content would change along the versions.